src/libs/actions/Report.ts

```diff
@@ -754,7 +754,7 @@ function navigateToAndOpenChildReport(childReportID = '0', parentReportAction: P
             parentReport?.policyID ?? CONST.POLICY.OWNER_EMAIL_FAKE,
             CONST.POLICY.OWNER_ACCOUNT_ID_FAKE,
             false,
-            '',
+            parentReport?.policyName ?? '',
```

Comment:
what does this fix?

Reply:
I'm not sure why we don't pass policyName of parent report when creating a child report. Please take a look. Subtitle becomes 'unavailable workspace'.

https://github.com/Expensify/App/assets/126839717/87727ffe-61a5-4f89-87e0-5a0c0509e13a

Reply:
oh we recently had lots of bug reports related to "Unavailable workspace".
So this change will fix this bug which long existed in production?

Reply:
Maybe

Reply:
I am not available to reproduce even without this change

Reply:
Sign out, sign in, and try again, please